includes/core/DataTransferObjects/class-data_transfer_object.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace WCPay\Core\DataTransferObjects;
+
+use ReflectionClass;
+use ReflectionException;
+
+abstract class Data_Transfer_Object
+{
+	/**
+	 * Return array of all properties.
+	 *
+	 * @return array
+	 */
+	public function to_array() {
+		$class = new ReflectionClass($this);
+		$data = [];
+		foreach ($class->getProperties() as $reflectionProperty) {
+			$data[$reflectionProperty->getName()] = $reflectionProperty->getValue();
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Change DTO property value if property is not private.
```

Comment:
This description could make a lot more sense, it seems like this changes a single property.

Reply:
Fixed.